src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
```

Comment:
The variable name 'result' could be more descriptive. Consider naming it 'styleObject' or something similar that indicates its purpose as a storage for CSS style declarations.